### PR TITLE
DRILL-6979: Added autofocus attribute to username on login page, and to query tex…

### PR DESCRIPTION
**Pull request by kfaraaz**
…tbox on Query tab

 @sohami @kkhatua  Can you please review ?

### CODE REVIEW

File: exec/java-exec/src/main/resources/rest/query/query.ftl [hunk 1]
```diff
@@ -74,7 +74,7 @@
       <div style="display: inline-block"><label for="query">Query</label></div>
       <div style="display: inline-block; float:right; padding-right:5%"><b>Hint: </b>Use <div id="keyboardHint" style="display:inline-block; font-style:italic"></div> to submit</div>
       <div id="query-editor-format"></div>
-      <input class="form-control" type="hidden" id="query" name="query"/>
+      <input class="form-control" type="hidden" id="query" name="query" autofocus/>
```
Comment (sohami) [inline exec/java-exec/src/main/resources/rest/query/query.ftl]: For some reason this doesn't seem to work. I can see cursor in query textbox but it's not focussed. I have to click the query textbox to type any query in it.

Comment (kfaraaz) [reply exec/java-exec/src/main/resources/rest/query/query.ftl]: Made the change, please review again.

Thanks.

File: exec/java-exec/src/main/resources/rest/login.ftl [hunk 2]
```diff
@@ -24,15 +24,15 @@
 <#macro page_body>
     <div class="page-header"></div>
     <div align="center" class="table-responsive">
-        <form role="form" name="input" action="/j_security_check" method="POST">
+        <form role="form" name="input" autocomplete="off" action="/j_security_check" method="POST">
```
Comment (sohami) [inline exec/java-exec/src/main/resources/rest/login.ftl]: Please revert back `autocomplete="off"`

Comment (kfaraaz) [reply exec/java-exec/src/main/resources/rest/login.ftl]: Due to a privacy concern, I would prefer that the browser not remember the values for such fields that may contain sensitive information (for example, a unique government identifier or credit card security code), even if the browser's autocomplete feature is enabled. 
Hence, autocomplete="off"

Comment (kfaraaz) [reply exec/java-exec/src/main/resources/rest/login.ftl]: Done, reverted autocomplete="off" as it is not fully supported by browsers. Please review.

**Comment by HanumathRao** [comment]
@kfaraaz  Can you please rebase onto the latest master and also squash the commits.

**Comment by kfaraaz** [comment]
I have squashed and rebased the changes. 

Thanks
Khurram

**Pull request merged**